person/views.py

```diff
@@ -78,16 +78,31 @@ def get_context_data(self, **kwargs):
                 command_chain = {}
 
                 last_cited = repr(membership.object_ref.lastciteddate.get_value().value)
-                node_list_raw = get_org_hierarchy_by_id(org.uuid, when=last_cited)
-                edge_list = get_command_edges(org.uuid, when=last_cited)
+                node_list_raw = get_chart_nodes(org.uuid, when=last_cited)
+                edge_list = get_chart_edges(org.uuid, when=last_cited)
+
+                # Create a "from" link for the person and their org.
+                edge_list.append({'from': str(org.uuid)})
+
+                # Push the person and their org in the node_list.
+                label = '<b>' + str(org.name) + '</b>' + '\n\n' + str(membership.object_ref.member.get_value().value.name)
+
+                person_on_detail = {
+                    'id': str(org.uuid),
+                    'label': str(label),
+                    'detail_id': ''
+                }
+
+                node_list_raw.append(person_on_detail)
 
                 # Add a unique URL to individual nodes for redirect to organization detail view
                 node_list = []
                 for node in node_list_raw:
-                    detail_id = node['detail_id']
-                    # Cast as a string to make it JSON serializable
-                    url = str(reverse_lazy('detail_organization', args=[detail_id]))
-                    node['url'] = url
+                    if node['detail_id']:
+                        detail_id = node['detail_id']
+                        # Cast as a string to make it JSON serializable
+                        url = str(reverse_lazy('detail_organization', args=[detail_id]))
```

Comment:
This is a super small nitpick, but I found out over the weekend that the `reverse` method avoids the lazy evaluation that is causing `reverse_lazy` to return an object instead of a string. I think if we use `reverse` here we can avoid casting to a string!

Reply:
Very good!